doc/docs/master/deploy-manage/manage/s3gateway/supported-operations.md

```diff
@@ -0,0 +1,456 @@
+
+* Create buckets: Creates a repo and branch.
+* Delete buckets: Deletes a branch or a repo with all branches.
+* List buckets: Lists all branches on all repos as S3 buckets.
+* Write objects: Atomically overwrites a file on a branch.
+* Remove objects: Atomically removes a file on a branch.
+* List objects: Lists the files in the HEAD of a branch.
+* Get objects: Gets file contents on a branch.
+
+## List Filesystem Objects
+
+If you have configured your S3 client correctly, you should be
+able to see the list of filesystem objects in your Pachyderm
+repository by running an S3 client `ls` command.
+To list filesystem objects, complete the following steps:
+
+1. Verify that your S3 client can access all of your Pachyderm repositories:
+
+   * If you are using MinIO, type:
+
+     ```bash
+     mc ls local
+     ```
+
+     **System Response:**
+
+     ```bash
+     [2019-07-12 15:09:50 PDT]      0B master.train/
+     [2019-07-12 14:58:50 PDT]      0B master.pre_process/
+     [2019-07-12 14:58:09 PDT]      0B master.split/
+     [2019-07-12 14:58:09 PDT]      0B stats.split/
+     [2019-07-12 14:36:27 PDT]      0B master.raw_data/
+     ```
+
+   * If you are using AWS, type:
+
+     ```bash
+     aws --endpoint-url http://localhost:30600 s3 ls
+     ```
+
+     **System Response:**
+
+     ```bash
+     2019-07-12 15:09:50 master.train
+     2019-07-12 14:58:50 master.pre_process
+     2019-07-12 14:58:09 master.split
+     2019-07-12 14:58:09 stats.split
+     2019-07-12 14:36:27 master.raw_data
+     ```
+
+   * If you are using S3cmd, type:
+
+     ```bash
+     s3cmd ls
+     ```
+
+     **System Response:**
+
+     ```bash
+     2019-07-12 15:09 master.train
+     2019-07-12 14:58 master.pre_process
+     2019-07-12 14:58 master.split
+     2019-07-12 14:58 stats.split
+     2019-07-12 14:36 master.raw_data
+     ```
+
+1. List the contents of a repository:
+
+   * If you are using MinIO, type:
+
+     ```bash
+     mc ls local/master.raw_data
+     ```
+
+     **System Response:**
+
+     ```bash
+     [2019-07-19 12:11:37 PDT]  2.6MiB github_issues_medium.csv
+     ```
+
+   * If you are using AWS, type:
+
+     ```bash
+     aws --endpoint-url http://localhost:30600/ s3 ls s3://master.raw_data
+     ```
+
+     **System Response:**
+
+     ```bash
+     2019-07-26 11:22:23    2685061 github_issues_medium.csv
+     ```
+
+   * If you are using S3cmd, type:
+
+     ```bash
+     s3cmd ls s3://master.raw_data/
+     ```
+
+     **System Response:**
+
+     ```bash
+     2019-07-26 11:22 2685061 s3://master.raw_data/github_issues_medium.csv
+     ```
+## Create an S3 Bucket
+
+You can create an S3 bucket in Pachyderm by using the AWS CLI or
+the MinIO client commands.
+The S3 bucket that you create is a branch in a repository
+in Pachyderm.
+
+To create an S3 bucket, complete the following steps:
+
+1. Use the `mb <host/branch.repo>` command to create a new
+S3 bucket, which is a repository with a branch in Pachyderm.
```

Comment:
This sentence doesn't exactly make sense because `mb <host/branch.repo>` is syntax specific to minio, whereas we have instructions for several different tools below

Reply:
fixed